README.rst

```diff
@@ -23,17 +23,18 @@ it... because paper.  I wrote this to make my life easier.
 How it Works
 ============
 
-1. Buy a document scanner like `this one`_ (used by me) or `this other one`_
+1. Paperless does not run your scanner.
```

Comment:
This is a good point, and worth mentioning, but as it's not really a "step", it shouldn't be numbered.  Instead, maybe start this section with a paragraph:

> Paperless does not control your scanner, it only helps you deal with what your scanner produces: